bedrock/etc/templates/firefox/retention/thank-you-a.html

```diff
@@ -0,0 +1,86 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-pebbles.html" %}
+
+{% block page_title %}
+  {{ _('Thank You') }}
+{% endblock %}
+
+{% block page_title_suffix %}{% endblock %}
+
+{% block page_desc %}
+  {{ _('Thank you page.') }}
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox-retention-thank-you-a' %}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main>
+  <canvas id="confetti-canvas"></canvas>
+  <section class="thank-you-section">
+    <div class="thank-you-header">
+      <div>It's all</div>
+      <div>thanks to you</div>
```

Comment:
Are these pages intended to be localized or English only? If the former, we'll need to wrap the strings, and probably do a little markup refactoring to handle line formatting in other locales. If the latter, what should happen if someone hits the URLs with a different locale, e.g. `fr`? We generally avoid showing English strings to other locales, but if the related campaign is only targeted at English speakers (meaning these URLs would generally not be discoverable by other locales), we can probably not worry about it.